utoipa-gen/src/path/parameter.rs

```diff
@@ -185,8 +185,8 @@ impl ToTokens for Parameter<'_> {
                 }
             }
 
-            if let Some(ref parameter_type) = parameter.parameter_type {
-                let property = Property::new(parameter_type.is_array, &parameter_type.ty);
+            if let Some(parameter_type) = &parameter.parameter_type {
+                let property = Property::new(parameter_type.clone());
```

Comment:
This clone could probably be removed if the type was reference or Cow. :) See above :point_down: